pymc/tests/distributions/test_timeseries.py

```diff
@@ -481,56 +481,120 @@ def test_change_dist_size(self):
         assert new_dist.eval().shape == (4, 3, 10)
 
 
-@pytest.mark.xfail(reason="Timeseries not refactored", raises=NotImplementedError)
-def test_GARCH11():
-    # test data ~ N(0, 1)
-    data = np.array(
+class TestGARCH11:
+    def test_logp(self):
+        # test data ~ N(0, 1)
+        data = np.array(
+            [
+                -1.35078362,
+                -0.81254164,
+                0.28918551,
+                -2.87043544,
+                -0.94353337,
+                0.83660719,
+                -0.23336562,
+                -0.58586298,
+                -1.36856736,
+                -1.60832975,
+                -1.31403141,
+                0.05446936,
+                -0.97213128,
+                -0.18928725,
+                1.62011258,
+                -0.95978616,
+                -2.06536047,
+                0.6556103,
+                -0.27816645,
+                -1.26413397,
+            ]
+        )
+        omega = 0.6
+        alpha_1 = 0.4
+        beta_1 = 0.5
+        initial_vol = np.float64(0.9)
+        vol = np.empty_like(data)
+        vol[0] = initial_vol
+        for i in range(len(data) - 1):
+            vol[i + 1] = np.sqrt(omega + beta_1 * vol[i] ** 2 + alpha_1 * data[i] ** 2)
+
+        with Model() as t:
+            y = GARCH11(
+                "y",
+                omega=omega,
+                alpha_1=alpha_1,
+                beta_1=beta_1,
+                initial_vol=initial_vol,
+                shape=data.shape,
+            )
+            z = Normal("z", mu=0, sigma=vol, shape=data.shape)
+        garch_like = t.compile_logp(y)({"y": data})
+        reg_like = t.compile_logp(z)({"z": data})
+        decimal = select_by_precision(float64=7, float32=4)
+        np.testing.assert_allclose(garch_like, reg_like, 10 ** (-decimal))
+
+    @pytest.mark.parametrize(
+        "arg_name",
+        ["omega", "alpha_1", "beta_1", "initial_vol"],
+    )
+    def test_batched_size(self, arg_name):
```

Comment:
Can we add implicit shape parametrization? 

If explicit_shape, the test works as now, otherwise it works by passing `shape=None` and `steps=steps`

Also suggesting a more explicit name for `arg_name`

```suggestion
    @pytest.mark.parametrize(
        "batched_param",
        ["omega", "alpha_1", "beta_1", "initial_vol"],
    )
    @pytest.mark.parametrize("explicit_shape", (True, False))
    def test_batched_size(self, explicit_shape, batched_param):
```